src/libPMacc/include/nvidia/functors/Assign.hpp

```diff
@@ -22,8 +22,6 @@
 
 #pragma once
 
-#include "pmacc_types.hpp"
```

Comment:
The `HDINLINE` used in this file is defined in `pmacc_types.hpp` so I think this is not superfluous.